test/powershell/Modules/Microsoft.PowerShell.Security/AclCmdlets.Tests.ps1

```diff
@@ -1,10 +1,64 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 Describe "Acl cmdlets are available and operate properly" -Tag CI {
-    It "Get-Acl returns an ACL object" -Pending:(!$IsWindows) {
+    It "Get-Acl returns an ACL DirectorySecurity object" -Pending:(!$IsWindows) {
```

Comment:
This might be better as `-Skip` if the ACL cmdlets are Windows-specific -- otherwise they may be pending for a long time.

Two other thoughts:

- I can't remember if there's a mechanism to skip a whole describe, but that might be easier here
- General thought: we should require some kind of in-code record of *why* a test is skipped or pending so that we can later re-evaluate

Reply:
Pester v5 can skip a whole Describe, but there is a lot of work to refactor tests to make things work in there.

Reply:
I think the Pester module used here is still not at v5.
```
Executing all tests in '.\test\powershell\Modules\Microsoft.PowerShell.Security\AclCmdlets.Tests.ps1' with Tags CI', 'Feature

Executing script .\test\powershell\Modules\Microsoft.PowerShell.Security\AclCmdlets.Tests.ps1
  [-] Error occurred in test script '.\test\powershell\Modules\Microsoft.PowerShell.Security\AclCmdlets.Tests.ps1' 0ms
    ParameterBindingException: A parameter cannot be found that matches parameter name 'Skip'.
    at <ScriptBlock>, F:\Source\PowerShell\test\powershell\Modules\Microsoft.PowerShell.Security\AclCmdlets.Tests.ps1: line 3
    at <ScriptBlock>, F:\Source\PowerShell\src\powershell-win-core\bin\Debug\net5.0\win7-x64\publish\Modules\Pester\4.10.1\Pester.psm1: line 1111
    at Invoke-Pester<End>, F:\Source\PowerShell\src\powershell-win-core\bin\Debug\net5.0\win7-x64\publish\Modules\Pester\4.10.1\Pester.psm1: line 1137
Tests completed in 174ms
Tests Passed: 0, Failed: 1, Skipped: 0, Pending: 0, Inconclusive: 0
```

I installed Pester 5.0.1 version and re-ran the test but the script download a different version of pester and performed the test.

Reply:
Yeah there's a lot we'd have to update here before we can move to v5 for this repo.